packages/database-editor/src/components/ImportWizard/pages/CreationPage.tsx

```diff
@@ -0,0 +1,67 @@
+import type { DatabaseTable } from '@axonivy/database-editor-protocol';
+import { Checkbox, Flex, Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from '@axonivy/ui-components';
+import { useTranslation } from 'react-i18next';
+import './CreationPage.css';
+
+export type CreationParameter = {
+  tableName: string;
+  enum?: boolean;
+  entity?: boolean;
+  repository?: boolean;
+};
+
+export type CreationPageProps = {
+  tables: Array<DatabaseTable>;
+  parameters: Array<CreationParameter>;
+  updateSelection: (table: string, key: Exclude<keyof CreationParameter, 'tableName'>, value: boolean) => void;
+};
+
+export const CreationPage = ({ tables, updateSelection, parameters }: CreationPageProps) => {
+  const { t } = useTranslation();
+
+  const checkState = (tableName: string, key: Exclude<keyof CreationParameter, 'tableName'>): boolean => {
+    const param = parameters.find(p => p.tableName === tableName);
+    if (!param) return false;
+    return param[key] === true;
+  };
+
+  return (
+    <Flex direction='column' className='import-page creation-page'>
+      <Table className='table-creation'>
```

Comment:
fine for now, as soon as you need more table features (like, sort, filter,...) I would use the tanstack table.